pipeline/inputs/mqtt.md

```diff
@@ -6,11 +6,11 @@ The **MQTT** input plugin, allows to retrieve messages/data from MQTT control pa
 
 The plugin supports the following configuration parameters:
 
-| Key | Description |
-| :--- | :--- |
-| Listen | Listener network interface, default: 0.0.0.0 |
-| Port | TCP port where listening for connections, default: 1883 |
-| Payload_Key |　Specify the key where the payload key/value will be preserved. |
+| Key         | Description                                                    | Default |
+| :---------- | :------------------------------------------------------------- | :------ |
+| Listen      | Listener network interface                                     | 0.0.0.0 |
+| Port        | TCP port where listening for connections                       | 1883    |
+| Payload_Key | Specify the key where the payload key/value will be preserved. |         |
```

Comment:
I put the default values in code markers to indicate values, and added a `_none_` to ensure we don't have a blank cell in the table.

```suggestion
| Listen      | Listener network interface. | `0.0.0.0` |
| Port        | TCP port where listening for connections. | `1883` |
| Payload_Key | Specify the key where the payload key/value will be preserved. | _none_ |
```

Reply:
Thanks!